services/handbooks/workspace/docs/domains/README.md

```
@@ -35,3 +35,45 @@
 3. Backend: `services/monolith/workspace/slices/{domain}/` を作成
 4. Frontend: `web/nyx/workspace/src/modules/{domain}/` を作成
 5. Proto: `proto/{domain}/v1/service.proto` を作成
+
+## Frontend Module Structure
+
+全 Frontend モジュールは以下の構造に従う:
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)